libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1022,10 +1022,75 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info2[] = {
     // @Range: 0 15
     AP_SUBGROUPINFO(link_quality, "LINK_Q", 1, AP_OSD_Screen, AP_OSD_Setting),
 
+    // @Param: CRSFPWR_EN
+    // @DisplayName: CRSFPWR_EN
+    // @Description: Displays the TX power when using the CRSF RC protocol
+    // @Values: 0:Disabled,1:Enabled
+
+    // @Param: CRSFPWR_X
+    // @DisplayName: CRSFPWR_X
+    // @Description: Horizontal position on screen
+    // @Range: 0 29
+
+    // @Param: CRSFPWR_Y
+    // @DisplayName: CRSFPWR_Y
+    // @Description: Vertical position on screen
+    // @Range: 0 15
+    AP_SUBGROUPINFO(crsf_tx_power, "CRSFPWR", 3, AP_OSD_Screen, AP_OSD_Setting),
```

Comment:
why index 3 and not 2?

Reply:
2 would collide with ``AP_GROUPINFO_FLAGS("ENABLE", 2, AP_OSD_Screen, enabled, 0, AP_PARAM_FLAG_ENABLE | AP_PARAM_FLAG_HIDDEN),``